ui/src/widgets/ui-notification/UINotification.vue

```diff
@@ -96,8 +96,12 @@ export default {
                 this.countdown = 100 - (elapsed / parseFloat(this.props.displayTime)) * 100
             }, 100)
         },
-        close () {
+        close (sendMsg) {
             this.show = false
+debugger
```

Comment:
```suggestion

```

Reply:
Hi @ZJvandeWeg,
I had not cleaned up my code yet, because I assume I will get some other feedback that I need to implement before it is being approved.